components/expenses/ExpensePayeeDetails.js

```diff
@@ -110,9 +110,15 @@ const ExpensePayeeDetails = ({ expense, host, isLoading, borderless, isLoadingLo
   const isPaid = expense?.status === expenseStatus.PAID;
   const displayedHost = expense?.host ?? host;
 
-  return isLoading ? (
-    <LoadingPlaceholder height={150} mt={3} />
-  ) : (
+  if (isLoading) {
+    <LoadingPlaceholder height={150} mt={3} />;
```

Comment:
This is missing a return, pushing a fix in https://github.com/opencollective/opencollective-frontend/pull/9000